kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -56,24 +56,46 @@ var ContentAreaView = BaseView.extend({
         this.$(".content").html("").append(view.$el);
     },
 
+    should_show_rating: function() {
+        /*
+        This function determines whether a rating should be shown for the content item.
+        returns: true or false
+        */
+        var entry_available = (typeof this.model !== "undefined") && !!this.model.get("available");
+        var logged_in = typeof window.statusModel.get("user_id") !== "undefined";
+        return logged_in && entry_available;
+    },
+
+    remove_rating_view: function() {
+        // Remove the rating view if it exists. Uses rating_view.close, so that the container element is not removed
```

Comment:
Still not sure I understand the rationale for this, given that we delete the reference to the view - do we now have a view without a reference lying around in the DOM?